niceman/retrace/tests/test_packagemanagers.py

```diff
@@ -42,6 +43,17 @@ def test_identify_packages():
 
 
 @skip_if(not apt)
+def test_utf8_file():
+    files = [u"/usr/share/ca-certificates/mozilla/TÜBİTAK_UEKAE_Kök_Sertifika_Hizmet_Sağlayıcısı_-_Sürüm_3.crt"]
+    # Simple sanity check that the pipeline works with utf-8
+    packages, origins, files = identify_packages(files)
+    pprint(files)
+    pprint(origins)
+    pprint(packages)
+    assert True
```

Comment:
could we actually check any output?  btw -- test probably should be skipped if ca-certificates package is not installed (or assert that we do not get some spurious result)